client/src/actions/map.js

```diff
@@ -0,0 +1,38 @@
+import { FETCH_MAP_MONUMENTS_PENDING, FETCH_MAP_MONUMENTS_SUCCESS, FETCH_MAP_MONUMENTS_ERROR } from '../constants';
+
+function fetchMonumentsPending() {
+    return {
+        type: FETCH_MAP_MONUMENTS_PENDING
+    };
+}
+
+function fetchMonumentsSuccess(monuments) {
+    return {
+        type: FETCH_MAP_MONUMENTS_SUCCESS,
+        payload: monuments
+    };
+}
+
+function fetchMonumentsError(error) {
+    return {
+        type: FETCH_MAP_MONUMENTS_ERROR,
+        error: error
+    };
+}
+
+/**
+ * Queries for all monuments, to be displayed on the monument map page
+ * This is an async action (redux-thunk)
```

Comment:
Gotta tell ya, redux does a whole lot of thinking

Reply:
really makes you thunk